crates/rpc-types-trace/src/otterscan.rs

```diff
@@ -3,12 +3,14 @@
 //! <https://www.quicknode.com/docs/ethereum/ots_getBlockTransactions>
 //! <https://github.com/otterscan/otterscan/blob/develop/docs/custom-jsonrpc.md>
 
-use alloy_primitives::{Address, Bloom, Bytes, TxHash, U256};
-use alloy_rpc_types_eth::{Block, Rich, Transaction, TransactionReceipt};
+use alloy_primitives::{Address, Bloom, Bytes, TxHash, B256, U256};
+use alloy_rpc_types_eth::{Block, Header, Rich, Transaction, TransactionReceipt, Withdrawal};
 use serde::{Deserialize, Serialize};
+use serde_repr::{Deserialize_repr, Serialize_repr};
 
 /// Operation type enum for `InternalOperation` struct
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize_repr, Deserialize_repr)]
+#[repr(u8)]
```

Comment:
I'd like to manually implement serde for this by doing match -> u8 -> string and String -> parse u8
so that we don't need the serde_repr dep

Reply:
sounds more reasonable, no need to introduce a new crate for only small things